deploy/docker/docker-entrypoint.sh

```diff
@@ -101,103 +101,114 @@ fi
 
 # Fix issue #42 - export env EMQX_DASHBOARD__DEFAULT_USER__PASSWORD to configure
 # 'dashboard.default_user.password' in etc/plugins/emqx_dashboard.conf
-if [[ ! -z "$EMQX_ADMIN_PASSWORD" ]]; then
+if [[ -n "$EMQX_ADMIN_PASSWORD" ]]; then
     export EMQX_DASHBOARD__DEFAULT_USER__PASSWORD=$EMQX_ADMIN_PASSWORD
 fi
 
-# echo value of $VAR_FULL_NAME hiding secrets if any
-echo_value () {
+# echo value of $VAR hiding secrets if any
+echo_value() {
     # get MASK_CONFIG
     MASK_CONFIG_FILTER="$MASK_CONFIG_FILTER|password|passwd|key|token|secret"
-    FORMAT_MASK_CONFIG_FILTER=$(echo $MASK_CONFIG_FILTER |sed -e "s/^[^A-Za-z0-9_]\{1,\}//g"|sed -e "s/[^A-Za-z0-9_]\{1,\}/\|/g")
+    FORMAT_MASK_CONFIG_FILTER=$(echo "$MASK_CONFIG_FILTER" | sed -r -e 's/^[^A-Za-z0-9_]+//' -e 's/[^A-Za-z0-9_]+$//' -e 's/[^A-Za-z0-9_]+/|/g')
 
     # check if contains sensitive value
-    if [ ! -z $(echo $(echo $VAR_NAME | tr '.' ' ') |grep -w -o -E "$FORMAT_MASK_CONFIG_FILTER") ]; then
+    if echo "$VAR_NAME" | grep -iqwE "$FORMAT_MASK_CONFIG_FILTER"; then
```

Comment:
Filter keyword ignore case.
`.` is also word splitter so they don't need to replace with space.
And use replace `! -z "$(grep -o ...)"` with `grep -q` because of https://github.com/koalaman/shellcheck/wiki/SC2143